linux/setup/linux_test_rescue.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Rescue tasks to be executed before common/test_rescue.yml
+# for collecting information from Linux guest OS 
+#
+- name: "Collect dmesg at test case failure"
+  include_tasks: ../utils/collect_dmesg.yml
+  vars:
+    dmesg_ignore_errors: true
+    dmesg_output_file_name: "dmesg_at_test_failure.log"
+
+- name: "Detect errors in dmesg output"
+  when:
+    - dmesg_output_file_path is defined
```

Comment:
"dmesg_output_file_path" has been initialized in above task. No need to check if it's defined.